app/routes/meetings/components/MeetingEditor.js

```diff
@@ -128,11 +138,43 @@ function MeetingEditor({
         </div>
 
         <Field
-          name="location"
-          label="Sted"
-          placeholder="Sted for møte"
-          component={TextInput.Field}
+          label="Bruk mazemap"
+          name="useMazemap"
+          component={CheckBox.Field}
+          fieldClassName={styles.metaField}
+          className={styles.formField}
+          value={useMazemap}
+          onChange={(e) => {
+            setUseMazemap(!useMazemap);
+          }}
+          normalize={(v) => !!v}
         />
+        {!useMazemap && (
+          <Field
+            name="location"
+            label="Sted"
+            placeholder="Sted for møte"
+            component={TextInput.Field}
+          />
+        )}
+        {useMazemap && (
+          <Flex alignItems="center">
+            <Field
+              label="Mazemap-rom"
+              name="mazemapPoi"
+              component={SelectInput.MazemapAutocomplete}
+              fieldClassName={styles.metaField}
+              placeholder="R1, Abakus, Kjel4"
+            />
+            {
+              // TODO: Bug; meeting is not updated when the field above is changed, so the link
+              // will go to whatever room mazemapPoi was set to when the editor was opened
+              /*meeting.mazemapPoi > 0 && (
+              <MazemapLink mazemapPoi={meeting.mazemapPoi} linkText="↗️" />
+            )*/
```

Comment:
In the editor for events, I've also added a small icon to link externally to mazemap for the given poi, but I couldn't make this work for meetings due to `meeting` not updating on every change to the field, like `event` does in the event edtior. I've left this as a comment for now.

Reply:
The reason for this is that the state of the form (both the event and the meeting, is stored in the `form` state in redux. This is extracted for each of the editors, so that we can use the current form values in the components again.

If you look in the `EventEditRoute`, in the `mapStateToProps`, you can see that the `event` prop consists of the current `event` **and** all the values from the event form overriding any properties. This is the `valueSelector` that is used.

You can also see in the `MeetingEditRoute` that the valueSelector is used to get the `invitingUsers`. So if you want to use the current `mazeMapPoi` form value, just override that prop on the `meeting` object with the one from the valueSelector just as you have done with events. Then this should work great :smile: